bundles/org.openhab.binding.omnilink/src/main/resources/OH-INF/thing/unit.xml

```diff
@@ -206,6 +210,10 @@
 		<label>Switch</label>
 		<description>Turn this unit on/off.</description>
 		<category>Switch</category>
+		<tags>
+			<tag>Control</tag>
+			<tag>Light</tag>
+		</tags>
```

Comment:
As above, does this only switch lights on and off as it will place a count of how many lights are turned on in the location tabs as per my long comment.

Reply:
Yes same as above, but this is only for simple on/off with no dimmer functions for switches that don't support dimming

Reply:
As above comment. If you think it needs to be tagged then feel free to resolve this.